corehq/form_processor/tests/utils.py

```diff
@@ -170,6 +172,10 @@ def _delete_all(db, viewname, **view_kwargs):
 )
 
 
+def sql_backend_case(cls):
```

Comment:
Hmm. yeah it looks like it picks up anything matching `(?:\b|_)[Tt]est`. Could either pass a specified matcher to nose or use something without test. maybe `ensure_sql_backend` although I'm not in love with that either.